catalog/core/catalog-core-camelcomponent/src/main/java/ddf/camel/component/catalog/content/ContentProducer.java

```diff
@@ -78,7 +101,14 @@ public void process(Exchange exchange)
 
         Message in = exchange.getIn();
         Object body = in.getBody();
-        File ingestedFile = null;
+        File ingestedFile;
+
+        LOGGER.debug("Headers : {} ",
```

Comment:
Might as well swap these lines so you can do `LOGGER.debug("Headers : {}", headerMap);`